docs/recipes/debugging-with-webstorm.md

```diff
@@ -13,6 +13,8 @@ In the `JavaScript file` field specify the path to AVA in the project's `node_mo
 
 In the `Application parameters` pass the CLI flags you're using and the test files you would like to debug, for example `--verbose test.js`.
 
+In the `Node parameters` pass the `--inspect-brk` flag to enable the Node inspector or `--inspect-brk=port` if you want to listen it on a specific port.
```

Comment:
Probably will be better if text `if you want to listen` will be in the start of sentence, because it is advanced and optional instruction (english is not my native language, so, I don't known how much it is relevant).